src/components/dialog/examples/dialog-closing-actions.tsx

```diff
@@ -2,7 +2,36 @@ import { Component, h, State } from '@stencil/core';
 
 /**
  * Custom closing actions
+ *
+ * Action buttons in dialogs can be used to add a clear visual indication for
+ * the sighted users to realize that the dialog can be closed by pressing
+ * a button as well.
+ * This may sometimes be considered an unnecessary usage of action buttons for
+ * sighted users. Because majority of them users know that clicking or tapping
+ * outside the dialog closes it.
+ *
+ * Such buttons usually are labeled as `OK`, `Dismiss` or `Close`.
+ * :::tip
+ * When to use action buttons for simple "close" actions?
+ * - In fullscreen dialogs where clicking outside to close is hard.
+ * - When big dialogs are opened on phones, which make tapping outside hard for users.
+ * - When designing with accessibility in mind, and for those users who
+ * use screen readers to navigate the user interface.
```

Comment:
For accessibility, it feels like there should always be a close button, but it could be visually hidden… but finding the best solution for that is perhaps more work than we're willing to put in right now?

Reply:
I think so… but that's a good idea to add an invisible button